packages/odyssey-react/src/components/Status/Status.test.tsx

```diff
@@ -46,17 +46,4 @@ describe("Status", () => {
     expect(labelElement).toHaveTextContent(statusLabel)
     expect(labelElement).toBeVisible();
   });
-
-  it('should apply an aria role of `status` when the `prop.role` is set', () => {
-    const { getByTestId } = render(
-      <Status 
-        label={statusLabel}
-        descriptor={statusDescriptor}
-        role="status"
-      />
-    );
-    const statusElement = getByTestId('ods-status');
-
-    expect(statusElement).toHaveAttribute('role', 'status');
-  });
```

Comment:
consider changing the test selectors in this file to use the new static role you applied. That is: `getByRole("status")`. There is an example in my TextInput tests. This will help keep test coverage in place for the change you made within this PR and also reinforce a11y support long term.

Reply:
Is it necessary to keep this test now that `role` is a static html attribute and not a prop? For ex, TextInput doesn't check for the presence of appropriate classes.

Happy to add the test but want to have clarity on our threshold for testing static html.

Reply:
@edburyenegren-okta good question - Does the status role you applied provide critical behavior? I believe it does for a11y so we'd want at least some amount of test coverage for it, even though it isn't dynamic via props yes.

Going further, this allows us to write better tests because we can do away with code like `data-testid` and instead focus on our assertions on ways that real users will interact with our components (a11y, labels, etc)

Reply:
Test added! I left @arnoldsandoval-okta's other tests as-is and treated `expect(getByRole("status")).toBeInTheDocument;` as an explicit a11y check.